### ISSUE TITLE
exey.io

### ISSUE THREAD

**Issue by gototheskinny** [bogachenko/fuckfuckadblock#285]
### Basic information

**Check the link:** `https://exey.io/NXNKyy1`
**Category:** Anti-adblock
**Reported from:** Turkey

### User environment

**Browser:** Chrome 99.0.4844.74
**Extension or program:** uBlock Origin
**DNS:** No, Tried with Adguard DNS, same result
**Extensions:** uBlock Origin, Dark Reader, SponsorBlock, Privacy Badger, FastForward

### Additional message

First of all I tried with Adguard dns enabled didn't work, disabled didn't work either. I think this is a side site of exe.io bc it forwards to this exey.io site just FYI.

**Comment by bogachenko** [comment]
Some kind of broken site, even with the adblock turned off, shows a usage notification.
***
Duplicate of #181

**Comment by gototheskinny** [comment]
Oh ok thank you I didn't know.

**Comment by bogachenko** [comment]
They updated the site, now there is no adblock. Check for yourself @gototheskinny

**Comment by gototheskinny** [comment]
With Adguard DNS, Peter Lowe’s Ad and tracking server list,Fuck Fuckadblock list, breaks this sites anti-adblock blocker. I tried every filter list and this filters broke this site. ublock and adguard annoyances works perfect.

**Comment by bogachenko** [comment]
Strange, I can not reproduce. How do you use the filter DNS server (which blocks `pagead2.googlesyndication.com`, when it is needed to unblock the site, for eg https://github.com/uBlockOrigin/uAssets/commit/97dc7adc676431f64cb19fcef6b9cd01d56ea139#diff-03b955eebef8886aa0f1ef70b305e9b7996de9a9e0adc9fe002db5a21d301c9fR15013 or https://github.com/AdguardTeam/AdguardFilters/commit/adf9fede24db197559ff8cc9571b575646300f7d#diff-beb9872d5fbb0d321b23246ef65d99c4d842cd31f5654869bb0974cbd0b97dc4R749-R752), do you use proxy/VPN or what (to deceive a specific site or application)?

**Comment by gototheskinny** [comment]
I checked again and now it doesn't work with AdGuard dns not the filter but the DNS address service. It did worked like I said. Its not working when I disable Ubo and adguard annoy, change the dns to my ips's dns (Turkey,Turk Telekom) and only select this Ffa list it's not working. There is a problem on this list with this particular site. When I select only ubo and adguard annoy separately and both it works without adguard dns of course. Now With adguard dns it's not working no matter how.

**Comment by bogachenko** [comment]
After 5 minutes, update, check again and let me know.
And additionally, please press the keyboard shortcut, you have chrome, right?, CTRL+SHIFT+C, go to the "Network" tab and try to find googlesyndication in the search. Because I thought you used vpn and decided to try it too, and the requests differ from my real ip from the ip that vpn gives

**Comment by gototheskinny** [comment]
Today i rechecked it and it's working even if i only use this filter. But with Adguard DNS is still causing issues. I did't use any VPN or Proxy or any hosts file only tried with Adguard DNS but i disabled to test this issue of course. Thank you for fixing this issue on your end.

**Comment by bogachenko** [comment]
yes, well, as I said earlier, you cannot use both an ad blocker and bloking by dns. Because if one program allows a domain, and the other, on the contrary, blocks it, then conflicts arise. Blocking by DNS for **a computer** is a rudiment.

**Comment by bogachenko** [comment]
@Yuki2718 It's time to remove obsolete rules

```
exey.io#@%#//scriptlet("set-constant", "randomVar", "true")
exey.io#@%#//scriptlet('prevent-fetch', 'pagead2.googlesyndication.com')
exey.io#@%#//scriptlet("set-constant", "randomVar", "true")
exey.io#@%#//scriptlet("set-constant", "disableItToContinue", "noopFunc")
exey.io#@%#//scriptlet('abort-current-inline-script', 'EventTarget.prototype.addEventListener', 'window.open')
exey.io#@%#//scriptlet("set-constant", "app_vars.force_disable_adblock", "undefined")
```
